src/ledgerwriter/pom.xml

```diff
@@ -32,7 +32,7 @@
     </parent>
 
     <properties>
-        <java.version>1.8</java.version>
+        <java.version>1.17</java.version>
```

Comment:
seeing this in the java unit tests : 
`[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.8.1:compile (default-compile) on project ledgerwriter: Fatal error compiling: error: invalid target release: 1.17 -> [Help 1]` 😢

Reply:
Realizing the runners are on Java 1.11, so I'm looking at bumping those!